jbpm-console-ng-case-mgmt/jbpm-console-ng-case-mgmt-client/src/main/java/org/jbpm/workbench/cm/client/actions/CaseActionsViewImpl.java

```diff
@@ -0,0 +1,123 @@
+
+import org.jboss.errai.common.client.dom.Div;
+import org.jboss.errai.common.client.dom.Event;
+import org.jboss.errai.common.client.dom.HTMLElement;
+import org.jboss.errai.ui.client.local.spi.TranslationService;
+import org.jboss.errai.ui.shared.api.annotations.DataField;
+import org.jboss.errai.ui.shared.api.annotations.EventHandler;
+import org.jboss.errai.ui.shared.api.annotations.ForEvent;
+import org.jboss.errai.ui.shared.api.annotations.Templated;
+import org.jbpm.workbench.cm.client.util.AbstractView;
+import org.jbpm.workbench.cm.model.CaseActionSummary;
+import org.uberfire.mvp.Command;
+
+@Dependent
+@Templated
+public class CaseActionsViewImpl extends AbstractView<CaseActionsPresenter> implements CaseActionsPresenter.CaseActionsView {
+
+    public static final String AVAILABLE_ACTIONS = "AVAILABLE";
+    public static final String INPROGRESS_ACTIONS = "IN_PROGRESS";
+    public static final String COMPLETED_ACTIONS = "COMPLETED";
+
+    @Inject
+    @DataField("available-actions")
+    CaseActionsListViewImp availableActions;
+
+    @Inject
+    @DataField("inprogress-actions")
+    CaseActionsListViewImp inprogressActions;
+
+    @Inject
+    @DataField("completed-actions")
+    CaseActionsListViewImp completedActions;
+
+    @Inject
+    @DataField("actions")
+
+    private Div actionsContainer;
+    private Command userTaskAddCommand;
+
+    @Inject
+    private TranslationService translationService;
+
+    @Override
+    public void init(final CaseActionsPresenter presenter) {
+        this.presenter = presenter;
+        availableActions.init(presenter);
+        inprogressActions.init(presenter);
+        completedActions.init(presenter);
+    }
+
+    @Override
+    public void setUserTaskAddCommand(Command userTaskAddCommand) {
+        this.userTaskAddCommand = userTaskAddCommand;
+    }
+
+
+    @Override
+    public HTMLElement getElement() {
+        return actionsContainer;
+    }
+
+    @EventHandler("AddTask")
+    public void onUserAddClick(@ForEvent("click") Event e) {
+        if (userTaskAddCommand != null) {
+            userTaskAddCommand.execute();
+        }
+    }
+
+    @Override
+    public void removeAllTasks() {
+        availableActions.removeAllTasks();
+        inprogressActions.removeAllTasks();
+        completedActions.removeAllTasks();
+    }
+
+    @Override
+    public void setAvailableActionsList(List<CaseActionSummary> caseActionList) {
+        availableActions.setCaseActionList(caseActionList);
+    }
+
+    @Override
+    public void setInProgressActionsList(List<CaseActionSummary> caseActionList) {
+        inprogressActions.setCaseActionList(caseActionList);
+    }
+
+    @Override
+    public void setCompletedActionsList(List<CaseActionSummary> caseActionList) {
+
+        completedActions.setCaseActionList(caseActionList);
+    }
+
+    @Override
+    public void updateListHeaders() {
+        availableActions.updateActionsHeader(translationService.format(AVAILABLE_ACTIONS),
```

Comment:
Seems like heatherText and tooltipTitle have always the same value, perhaps you could combine into a single parameter to avoid calling the translation service twice.